src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -188,18 +165,23 @@ private static String generateHeader(String pluginName){
      *
      * @return String  total documentation for the page
      */
-    public static String generatePluginHelp(String pluginName, Map<String, List<StepDescriptor>> byPlugin, boolean genHeader){
+    public static String generatePluginHelp(String pluginName, String displayName, Map<String, List<StepDescriptor>> byPlugin, boolean genHeader){
         Main.isUnitTest = true;
 
         //TODO: if condition
         StringBuilder whole9yards = new StringBuilder();
         if(genHeader){
-            whole9yards.append(generateHeader(pluginName));
+            whole9yards.append(generateHeader(displayName));
```

Comment:
Part of the reason the plugin name floated around in there was so the users would know what plugin specifically gave them what step.  I'm not sure if you want to keep that around somewhere or if they can figure it out from the nicer plugin displayName